src/aspire/estimation/covar2d.py

```diff
@@ -149,27 +153,29 @@ def get_covar(self, coeffs, ctf_fb=None, ctf_idx=None, mean_coeff=None,
             raise RuntimeError('The coefficients need to be calculated!')
 
         if (ctf_fb is None) or (ctf_idx is None):
-            return self._get_covar(coeffs, mean_coeff, do_refl)
+            ctf_idx = np.zeros(coeffs.shape[1], dtype=int)
+            ctf_fb = [blk_diag_eye(blk_diag_partition(RadialCTFFilter().fb_mat(self.basis)))]
 
         def identity(x):
             return x
 
-        if covar_est_opt is None:
-            covar_est_opt = {'shrinker': 'None', 'verbose': 0, 'max_iter': 250, 'iter_callback': [],
+        default_est_opt = {'shrinker': 'None', 'verbose': 0, 'max_iter': 250, 'iter_callback': [],
                              'store_iterates': False, 'rel_tolerance': 1e-12, 'precision': 'float64',
-                             'preconditioner': 'identity'}
+                             'preconditioner': identity}
+        covar_est_opt = fill_struct(default_est_opt, covar_est_opt)
```

Comment:
Switch `default_est_opt` and `covar_est_opt`.

Reply:
done.